examples/chainfleetautoscaler.yaml

```diff
@@ -27,24 +27,24 @@ kind: FleetAutoscaler
 metadata:
   name: chain-fleet-autoscaler
 spec:
+  fleetName: "fleet-example"
   policy:
     # Chain based policy for autoscaling.
     type: Chain
     chain:
       # Id of chain entry. If not set, the Id will be defaulted to the index of the entry within the chain.
       - id: "weekday"
-        type: Schedule  # Schedule based condition.
```

Comment:
🤔 why did type disappear? Should definitely still be there, since it's a include of `policy` within `chain`.

Reply:
I think the `type` you're referring to is for the denoting the policy type. However, currently the CRD doesn't have a type to denote whether or not an entry is scheduled or not, so I'll add that in the CRD and add it back to the example as well.

Reply:
(I think this is resolved now, it's down on new line 38)